local-tests/test_update.py

```diff
@@ -20,10 +21,16 @@
 
 def query_runtime_version(nodes):
     print('Current version:')
+    versions = []
```

Comment:
wdyt about making it set already instead of a list?

Reply:
Of course, why haven't I thought about that ^^'